bindings/java/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_subdirectory(com/google/iree/native)
```

Comment:
nit: add newline at the end of this file. You can configure your editor to do this automatically ([vscode](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files))

reasoning: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
> Therefore, lines not ending in a newline character aren't considered actual lines. That's why some programs have problems processing the last line of a file if it isn't newline terminated.

Reply:
Fixed